mu-plugins/blocks/global-header-footer/blocks.php

```diff
@@ -983,12 +983,15 @@ function set_current_item_class( $menu_items ) {
  * @return string
  */
 function get_menu_url_for_current_page( $menu_items ) {
+	$host    = strtolower( $_SERVER['HTTP_HOST'] );
+	$uri     = strtolower( $_SERVER['REQUEST_URI'] );
+	$compare = "https://{$host}{$uri}";
 
-	if ( 'translate.wordpress.org' === $_SERVER['HTTP_HOST'] ) {
+	if ( 'translate.wordpress.org' === $host ) {
 		return 'https://make.wordpress.org/';
 	}
 
-	if ( 'developer.wordpress.prg' === $_SERVER['HTTP_HOST'] ) {
+	if ( 'developer.wordpress.prg' === $host ) {
```

Comment:
I think this is a typo?

```suggestion
	if ( 'developer.wordpress.org' === $host ) {
```